stable/appmesh-inject/templates/deployment.yaml

```diff
@@ -15,6 +15,7 @@ spec:
       labels:
         app.kubernetes.io/name: {{ include "appmesh-inject.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/part-of: appmesh
```

Comment:
these changes seem unrelated to the e2e test addition. Consider placing these changes into a different PR.

Reply:
This is required change for the `teardown` function, having a common label on all App Mesh pods makes it possible to display the logs for debugging.

Reply:
ack, got it. thanks for the explanation!